src/Tabs.php

```diff
@@ -9,7 +9,7 @@
 class Tabs extends View
 {
     public $defaultTemplate = 'tabs.html';
-    public $ui = 'tabular menu';
+    public $ui = 'tabular menu stackable';
```

Comment:
I support this change in Tabs.

![image](https://user-images.githubusercontent.com/2228672/214054777-df01893b-5d43-4f0c-ac64-95d8e656a0a6.png)

there is an UX issue (the tab menu is hard it understand vs. other components/text) which should be reported, but I do not object to merge it until fixed

Reply:
Why not for menu? Please try a grid with larger horizontal menu: The row stack perfectly, but the menu does bot in current code. With stackable all fine, so I would merge both changes

Reply:
see https://github.com/atk4/ui/pull/1973#discussion_r1084107040

my primary comment on this PR was in reaction on "fix 1780" description, which is definitely not fixed by this PR

stackable in Menu might be ok in some cases, but UX can be visually very different and can take much more space which might be not wanted

test `demos/collection/grid.php` demo - when viewport width is below stackable treshold, the table is already very badly reformat, so I would actually prefer to `unstackable` there